### PR TITLE
Add web server stack for contacts admin

### PR DESCRIPTION
**Pull request by ChrisBAshton**
This is to facilitate https://trello.com/c/cIv5x8wB/2348-5-add-ability-to-unpublish-with-a-redirect-in-contacts-app.

Contacts Admin doesn't work perfectly with govuk-docker: notably, you currently need to run Publishing API in another shell. I'm not sure why that's the case, and am hoping it is something we can fix here, rather than have to document in https://github.com/alphagov/contacts-admin/pull/838.

### CODE REVIEW

**Review by brucebolt — APPROVED** [review]

**Review by callumknights — APPROVED** [review]

File: projects/contacts-admin/docker-compose.yml [hunk 1]
```diff
@@ -29,3 +29,22 @@ services:
...
+      - redis
+      - publishing-api-app
+      - nginx-proxy
+      - content-store-app
+      - whitehall-app
+    environment:
+      DATABASE_URL: "mysql2://root:root@mysql-5.5/contacts_development"
+      TEST_DATABASE_URL: "mysql2://root:root@mysql-5.5/contacts_test"
+      REDIS_URL: redis://redis
+      VIRTUAL_HOST: contacts-admin.dev.gov.uk
```
Comment (benthorner) [inline projects/contacts-admin/docker-compose.yml:45]: I'm a bit late to the party, but trying this out locally breaks due to [this issue](https://github.com/alphagov/govuk-docker/issues/176). Do we need a separate PR to fix that?

Comment (ChrisBAshton) [reply projects/contacts-admin/docker-compose.yml:45]: That PR is here: https://github.com/alphagov/contacts-admin/pull/838

Comment (benthorner) [reply projects/contacts-admin/docker-compose.yml:45]: Ah, I see - I didn't clock that it was hidden in there, thanks!
